test_conformance/compiler/test_compiler_defines_for_extensions.cpp

```diff
@@ -80,8 +80,10 @@ const char *known_extensions[] = {
     "cl_khr_external_semaphore_opaque_fd",
     "cl_khr_external_semaphore_sync_fd",
     "cl_khr_command_buffer",
+    "cl_khr_command_buffer_mutable_dispatch",
     "cl_khr_external_memory",
     "cl_khr_external_memory_opaque_fd",
+    "cl_khr_extended_async_copies",
```

Comment:
I think [cl_khr_extended_async_copies](https://registry.khronos.org/OpenCL/specs/3.0-unified/html/OpenCL_Ext.html#cl_khr_extended_async_copies) shouldn't be considered an API-only extension.

Reply:
ahh, I missed the comment on line 54 differentiating the types of extensions. Will update